pagefaultgames/pokerogue
# [Misc] Remove unnecessary text on summary_bg.png

## What are the changes?
Modify 1 UI image to resolve small visual glitch on Korean locale.

## Why am I doing these changes?
Shortly, screenshots can say everything.

In `summary-ui-handler.ts`, `summaryLabel` and `tabSprite` draws text.
But `summary_bg.png` draws them one more time.

Alphabet glyph of `pokemon-emerald-pro.ttf`(3rd Gen) and  `PokePT_Wansung.ttf`(4th Gen: no 3rd in Kor) are not same.
With double drawing, no visible problem in English, but graphic glitch with Korean.

If synchronizing alphabet glyphs is better way, just close it.
I'll update font one more time. (maybe attached that to #1678)

## What did change?
In Default theme pokemon summary, there is no more extra hidden texts.
No effect with Legacy.

### Screenshots/Videos
![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/48ed4bcf-7365-421b-9545-6f6ab136a69e)
Old one. We can see text is drawn twice.

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/c019d00f-cc62-49ae-ac27-34aad982fb4a)
In other language, text is drawn twice but they're same. No visible problem.

![image](https://github.com/pagefaultgames/pokerogue/assets/87186129/df64dfde-66f7-4bdb-8f71-69c9aa332153)
New one. No overlapping text, no glitch.

## How to test the changes?
Open pokemon summary window in battle.
Swipe left and right to check all of status, stats and moves.

## Checklist
